samples/tfm_integration/psa_protected_storage/sample.yaml

```diff
@@ -0,0 +1,22 @@
+sample:
+  description: Protected Storage API sample
+  name: PSA Protected Storage
+common:
+  tags: psa
+  platform_allow: mps2_an521_nonsecure v2m_musca_s1_nonsecure
+                    nrf5340dk_nrf5340_cpuappns nrf9160dk_nrf9160ns bl5340_dvk_cpuappns
+  harness: console
+  harness_config:
+    type: multi_line
+    regex:
+      - "TF-M Protected Storage sample started."
```

Comment:
I think "TF-M" is not printed

Reply:
Ah, that looks to be the troublemaker, yes. Thanks!